var/spack/repos/builtin/packages/flux-core/package.py

```diff
@@ -14,7 +14,13 @@ class FluxCore(AutotoolsPackage):
     url      = "https://github.com/flux-framework/flux-core/releases/download/v0.8.0/flux-core-0.8.0.tar.gz"
     git      = "https://github.com/flux-framework/flux-core.git"
 
-    version('master',  branch='master')
+    version('master', branch='master')
+    version('0.11.3',
+            sha256='91b5d7dca8fc28a77777c4e4cb8717fc3dc2c174e70611740689a71901c6de7e',
+            preferred=True,
```

Comment:
`preferred=True` is not needed here, Spack already defaults to the latest stable numbered release

Reply:
Good catch! I forgot to remove that after I pull v0.12 and v0.13.  Let me update the title and OP of this PR.